src/main/java/org/terasology/metalrenegades/ai/system/CitizenSpawnSystem.java

```diff
@@ -39,10 +40,16 @@
 import org.terasology.metalrenegades.economy.MarketCitizenComponent;
 import org.terasology.metalrenegades.economy.TraderComponent;
 import org.terasology.metalrenegades.economy.actions.ShowTradingScreenAction;
+import org.terasology.metalrenegades.minimap.events.AddCharacterToOverlayEvent;
+import org.terasology.namegenerator.creature.CreatureAssetTheme;
+import org.terasology.namegenerator.creature.CreatureNameProvider;
 import org.terasology.registry.In;
 
 import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Random;
+
+import static org.terasology.namegenerator.creature.CreatureAssetTheme.COUNTRY;
```

Comment:
As I've requested to change the name list prefix to "Old West" this (and probably other spots) need changes now... 🙄 I'm eager to merge https://github.com/Terasology/NameGenerator/pull/23, though (after renames) 👍 

```suggestion
import static org.terasology.namegenerator.creature.CreatureAssetTheme.OLD_WEST;
```